api/managementpb/checks.proto

```diff
@@ -33,10 +33,35 @@ message SecurityCheckResult {
   map<string, string> labels = 4;
 }
 
+// SecurityCheckState contains check name and status.
+message SecurityCheckState {
+  string name = 1;
+  bool disabled = 2;
+}
+
+// SecurityCheckParams specifies check parameters.
+message SecurityCheckParams {
```

Comment:
`ChangeSecurityCheckParams` (method name + params), again, like in https://github.com/percona/pmm/blob/PMM-2.0/api/inventorypb/agents.proto

Reply:
Done